docs/index.rst

```diff
@@ -1,12 +1,16 @@
+Welcome to |project| documentation!
```

Comment:
**suggestion** This reads a bit strange to me, let me suggest a change:

```suggestion
Welcome to the |project| documentation!
```

Reply:
I'm not necessarily opposed to this change, but do consider that by customizing this file, you're now diverging from what the skeleton is doing for every other project and adding a conflict if this line (or an adjacent one) is ever changed in the skeleton. I usually will compromise the precision of each downstream project in order to manage these concerns in one place (the skeleton). I'd ask yourself, if this language is awkward for `kustomize`, is it also awkward for `tempora` or `importlib_metadata`. If it's awkward for all, it would be preferable to propose the change in skeleton so it can apply to all. I'll leave it as is, but wanted to share this perspective.